test/integration/extension.test.ts

```diff
@@ -36,9 +36,29 @@ import {
 	isVendorSupported
 } from '../../src/util';
 
+function queryDefaultGopathSync(): string | null {
+	const goExecutable = getBinPath('go');
+	if (!goExecutable) {
+		console.warn(`Failed to run "go env GOPATH" to find mod file as the "go" binary cannot be found`);
+		return null;
+	}
+	let gopath: string;
+	try {
+		const stdout = cp.execFileSync(goExecutable, ['env', 'GOPATH']);
+		console.log(`Got go env GOPATH result: ${stdout}`);
+		gopath = stdout.toString().trim();
+	} catch (err) {
+		console.warn(`Error when running go env GOPATH: ${err}`);
+	}
+	return gopath;
+}
+
 suite('Go Extension Tests', function() {
 	this.timeout(10000);
-	const gopath = getCurrentGoPath();
+	let gopath = getCurrentGoPath();
+	if (!gopath) {
+			gopath = queryDefaultGopathSync();
+	}
```

Comment:
Try the below after updating the callback function to be async

```suggestion
	await updateGoPathGoRootFromConfig();
```

Reply:
That's what I tried before but failed to convince the test suite to wait - the test just terminates 
 without running actual tests and declares the success. Need help with the Mocha suite setup.

Reply:
`suite()` cannot accept an async function, so you'll need to declare `gopath`, `repoPath`, `fixturePath`, etc. with `let` instead of `const`, and assign them in `suiteSetup()`, which can take async functions.

Reply:
Thanks @zmb3. Made the suggested change.